examples/helpers/mail/mail_example.py

```diff
@@ -124,10 +124,11 @@ def build_attachment4_s3_store_custom_session():
     attachment.disposition = "attachment"
     attachment.content_id = "S3Session"
     attachment.bucket = YOUR_BUCKET_NAME
-    attachment.session =    (aws_access_key_id=YOUR_ACCESS_KEY,
-                            aws_secret_access_key=YOUR_SECRET_KEY,
-                            aws_session_token=SESSION_TOKEN,
-                            region_name=REGION_NAME)
+    attachment.session =    boto3.session.Session(
+                                    aws_access_key_id=YOUR_ACCESS_KEY,
+                                    aws_secret_access_key=YOUR_SECRET_KEY,
+                                    aws_session_token=SESSION_TOKEN,
+                                    region_name=REGION_NAME)
```

Comment:
The spacing is an issue here, PEP8 warning for multiple spaces after the `=` operator. Something like this would fix the issue.

```python
attachment.session = boto3.session.Session(
    aws_access_key_id=YOUR_ACCESS_KEY,
    aws_secret_access_key=YOUR_SECRET_KEY,
    aws_session_token=SESSION_TOKEN,
    region_name=REGION_NAME
)

```

Reply:
You are right, I will go ahead and incorporate it.